docs/src/main/asciidoc/native-and-ssl.adoc

```diff
@@ -35,10 +35,17 @@ If you open the application's configuration file (`src/main/resources/applicatio
 
 [source,properties]
 ----
-quarkus.rest-client."org.acme.restclient.CountriesService".url=https://restcountries.eu/rest
+quarkus.rest-client."org.acme.rest.client.ExtensionsService".url=https://stage.code.quarkus.io/api
 ----
 which configures our REST client to connect to an SSL REST service.
 
+For the purposes of this guide, we also need to remove the configuration that starts the embedded WireMock server that stubs REST client responses so the tests actually propagate calls to the https://stage.code.quarkus.io/api. Update the test file `src/test/java/org/acme/rest/client/ExtensionsResourceTest.java` and remove the line:
```

Comment:
I don't get the motivation for removing wiremock

Reply:
Wiremock server before this was wrong and propagated actual calls from tests to the restcountries.eu. This native SSL guide relied on this mistake which once fixed couldn't reproduce this issue that this guide was referring to as stubs worked :).

Reply:
Maybe instead of showing the issue in tests, the guide could just mention that when someone tries it out, it fails?

Removing wiremock seems a bigger change to the project, and might send a message to not mock the API if it's https

Reply:
Agreed, but I will do that in a subsequent PR. This is exactly why I didn't want to do broader refactoring under the "Update REST client guides..." commits :)